src/pretix/presale/templates/pretixpresale/event/fragment_cart.html

```diff
@@ -4,331 +4,353 @@
-                One product
-            {% plural %}
-                {{ num }} products
-            {% endblocktrans %}
-        </span>
-    </div>
-    <div class="count hidden-xs hidden-sm">
-    </div>
-    <div class="col-md-3 col-xs-6 col-md-offset-3 price">
-        <strong>{{ cart.total|money:event.currency }}</strong>
+</div>
 
-        {% if not event.settings.display_net_prices and cart.tax_total and not editable %}
-            <small class="text-muted">
-                {% blocktrans trimmed with tax_sum=cart.tax_total|money:event.currency %}
-                    incl. {{ tax_sum }} taxes
+<div class="row">
+    <div class="col-md-12">
+    {% if not cart.is_ordered %}
+        <p class="text-muted" id="cart-deadline" data-expires="{{ cart.first_expiry|date:"Y-m-d H:i:sO" }}">
+            {% if cart.minutes_left > 0 or cart.seconds_left > 0 %}
+                {% blocktrans trimmed with minutes=cart.minutes_left %}
+                    The items in your cart are reserved for you for {{ minutes }} minutes.
                 {% endblocktrans %}
-            </small>
-        {% endif %}
-
-        {% if editable and vouchers_exist and not cart.all_with_voucher %}
-            <br>
-            <a class="js-only apply-voucher-toggle" href="#">
-                <span class="fa fa-tag" aria-hidden="true"></span> {% trans "Redeem a voucher" %}
-            </a>
-            <form action="{% eventurl event "presale:event.cart.voucher" cart_namespace=cart_namespace|default_if_none:"" %}"
-                  data-asynctask-headline="{% trans "We're applying this voucher to your cart..." %}"
-                  method="post" data-asynctask class="apply-voucher">
-                {% csrf_token %}
-                <label for="voucher_code" class="sr-only">{% trans "Voucher code" %}</label>
-                <div class="input-group">
-                    <input type="text" class="form-control" name="voucher" id="voucher_code" placeholder="{% trans "Voucher code" %}" aria-label="{% trans "Voucher code" %}">
-                    <span class="input-group-btn">
-                        <button class="btn btn-primary" type="submit">
-                            <span class="fa fa-check" aria-hidden="true"></span><span class="sr-only"> {% trans "Redeem voucher" %}</span>
-                        </button>
-                    </span>
-                </div>
-            </form>
+            {% else %}
+                {% trans "The items in your cart are no longer reserved for you. You can still complete your order as long as they’re available." %}
+            {% endif %}
+        </p>
+    {% else %}
+        <p class="sr-only" id="cart-deadline">
+        {% if editable %}
+            {% trans "Overview of your ordered products. You can still change your order." %}
+        {% else %}
+            {% trans "Overview of your ordered products. You cannot change your order anymore." %}
         {% endif %}
```

Comment:
The combination of `is_ordered` and `editable` can never happen. If orders are change-able, this will be in different ways through cancellation / rebook buttons. I'd suggest just

```suggestion
        {% trans "Overview of your ordered products." %}
```

Reply:
I changed the surrounding HTML a bit, too, but added your suggestion.